### ISSUE TITLE
Expected identifier error in IE8

### ISSUE THREAD

**Issue by kevana** [gaearon/redux-thunk#45]
At line 6:
```
exports.default = thunkMiddleware;
```

`default` is a reserved keyword in IE8 and it throws an errror when dot notation is used. it does work when bracket notation is used:
```
exports['default'] = thunkMiddleware;
```

I think this is a bug in the upstream babel-plugin-add-module-exports, but wanted to show it here in case people are looking here first.

**Comment by kevana** [comment]
Nevermind, problem with my webpack config....

**Comment by gaearon** [comment]
Please let us know if it doesn't get resolved soon and we need to work around it.

**Comment by kevana** [comment]
Will do. I'm using the string-replace-webpack-plugin to work around it for now.

**Comment by gaearon** [comment]
This should be fixed in 4.0.6.
Please verify.

**Comment by gaearon** [comment]
Sorry, wrong repo.

**Comment by gaearon** [comment]
Should be fixed in 1.0.3.
Please verify?

**Comment by kevana** [comment]
Verified fixed, thanks.
